server/src/api/api.graphql

```diff
@@ -39,16 +45,16 @@ type Item {
 	totalAvailable: Int!
 	# The largest quantity of this item that can be requested in one request
 	maxRequestQty: Int!
-	# The value of this item to replace it if lost or damaged
-	price: Float!
+    # The value of this item to replace it if lost or damaged (only visible to admins)
```

Comment:
would prefer if admin related fields were grouped (nit)

Reply:
Grouped as in adjacent to each other or in some kind of sub-object?

Reply:
adjacent

Reply:
Yeah I can do that